docs/README.md

```diff
@@ -124,17 +132,40 @@ productSearch(
     If you want faster searches and do not care about most up to date prices and promotions, use skip value.
     """
     simulationBehavior: SimulationBehavior = default
+    """
+    If true, all the products will be solved by VTEX API.
+    """
+    productOriginVtex: Boolean = false
+    """
+    Indicates how the search-engine will deal with the fullText.
+    """
+    operator: Operator
+    """
+    Indicates how the search engine will correct misspeled words.
+    """
+    fuzzy: String
+    """
+    It is similar to fuzzy and operator but is used for scenarios where fuzzy and operator are not enough.
+    """
+    searchState: String
```

Comment:
Can you give examples of possible values to these fields `operator`, `fuzzy` and `searchState`?